Authenticator/authenticator.py

```diff
@@ -0,0 +1,39 @@
+from jupyterhub.auth import Authenticator
+from tornado import gen
+import csv
+import logging
+
+
+class MyAuthenticator(Authenticator):
+    @gen.coroutine
+    def authenticate(self, data, handler):
+
+        logger = logging.getLogger(__name__)
```

Comment:
There is already a logger accessible here as `self.log`, so we don't need to configure a new logger.

Reply:
Done!

Reply:
@minrk  I have updated it